### PR TITLE
First draft of simplified README based on dplyr README

### LINKED ISSUE #192: Consolidate basic articles
1. dm.Rmd
2. dm-class-and-basic-operations.Rmd
3. dm-setup.Rmd

- [ ] There's a lot of redundancy in 2 and 3, maybe merge into one text?
- [ ] 1 is the first text that potential users will see after the README if they are interested. It needs to highlight the most important topics and link to extended comprehensive articles. Currently it emphasizes some features that seem secondary.
- [ ] Review all other articles, ensure that they fit together and each concept is described only once in detail and linked from the other places where it's used
- [ ] Think about README, ensure that README is consistent with the rest.
- [ ] Do we need `index.md` (different README on GitHub and front page)?

### PR DESCRIPTION
**Pull request by jawond**
Per conversation with @krlmlr. It's an extreme change, and the first step in streamlining and consolidating the documentation.

Closes #192.

### CODE REVIEW

**Comment by krlmlr** [comment]
Added FIXMEs and rough structure, as discussed today. Feel free to adapt. We'll stash away unused content later.

**Comment by krlmlr** [comment]
Thanks!

File: vignettes/dm.Rmd [hunk 1]
```diff
@@ -51,17 +52,37 @@ fin_dm
...
+count(fin_dm$trans)
+```
+
+Here we focus on the `loans`, `accounts`, `districts` and `trans` tables:
+
+```{r select}
+fin_dm_small <- fin_dm[c("loans", "accounts", "districts", "trans")]
+```
+
+The `dm_select_tbl()` verb offers a way to do the same in a `%>%` pipe.
```
Comment (jawond) [inline vignettes/dm.Rmd:72]: #FIXME example code demonstrating this function

Comment (krlmlr) [reply vignettes/dm.Rmd:72]: Thanks, in af43569.

I'm unsure if introducing the concept that "dm behaves as a list" in passing is a good idea.

**Pull request merged**